unlock-app/src/services/walletService.js

```diff
@@ -25,15 +25,17 @@ export const keyId = (lock, owner) => [lock, owner].join('-')
  * actually retrieving the data from the chain/smart contracts
  */
 export default class WalletService extends EventEmitter {
-  constructor(availableProviders = providers) {
+  constructor({ providers, runningOnServer, unlockAddress } = configure()) {
     super()
-    this.providers = availableProviders
+    this.unlockAddress = unlockAddress
```

Comment:
Move the `Web3Utils.toChecksumAddress` part from below up here so we do not have both `this.unlockAddress` and `this.unlockContractAddress`.